riscv/encoding.h

```diff
@@ -228,6 +228,10 @@
 
 #define PTE_TABLE(PTE) (((PTE) & (PTE_V | PTE_R | PTE_W | PTE_X)) == PTE_V)
 
+#define ENVCFG_CBIE    0x30
+#define ENVCFG_CBCFE   0x40
+#define ENVCFG_CBZE    0x80
```

Comment:
My changes including `CBIE`, `CBCFE` and `CBZE` bits are merged into `riscv-opcodes`.
https://github.com/riscv/riscv-opcodes/pull/94

Because `riscv/encoding.h` is intended to be auto-generated by `riscv-opcodes`, managing our bits (especially with *our* names) is not a good idea.

Reply:
OK. I'll update encoding.h.